ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -487,6 +485,8 @@ trait H2OGridSearchParams extends H2OCommonSupervisedParams {
     set(selectBestModelBy, validated)
   }
 
-
-  def setSelectBestModelDecreasing(value: Boolean): this.type = set(selectBestModelDecreasing, value)
```

Comment:
Shouldn't this be deprecated?

Reply:
I think at this particular change we are fine without deprecation as it's not really highly exposed API, but we can deprecated, sure

Reply:
done